dnf5-plugins/config-manager_plugin/addrepo.cpp

```diff
@@ -116,9 +96,11 @@ std::string generate_repoid_from_url(const std::string & url) {
     // Limits the length of the repository id.
     // Copies the first and last 100 characters. The substring in between is replaced by a crc32 hash.
     if (ret.size() > 250) {
-        std::string_view tmp{ret};
+        unsigned int sz = static_cast <unsigned int> (ret.size());
+        char middle[sz - 200]{};
+        ret.copy(middle, sz - 200, 100);
```

Comment:
Do we have to copy the substring instead of passing `ret.c_str() + 100` to crc32()?

Reply:
No, we don't.  I'll fix that.